data/core/encyclopedia/calendar.cfg

```diff
@@ -11,8 +11,9 @@
 [topic]
     id=..calendar
     title= _ "Calendar"
-    text= _ "Each year in the Wesnothian calendar is composed of 12 months, starting around early spring and ending in late winter. These are, in order:
+    text= _ "Each year in the Wesnothian calendar is composed of 12 or 13 months, starting at the beginning of spring and ending in late winter. Newmoon is added every three years to stay within the season, alternating between 29 and 30 days. Scatterseed is 29 days long, and all following months alternate between 29 and 30 days to stay in line with the lunar cycle. These months are, in order:
```

Comment:
An entire extra month seems a little excessive to me. Are there existing (either actively in use or historical) calendars that do this?

Reply:
According to Wikipedia, the Chinese Calendar does this. It is a common feature of some lunisolar calendars.

Reply:
Having leap-months added to the calendar feels like too much detail, and it's the sort of complication that will trip up campaign authors if they use the calendar but forget about the special years; or if someone writes a campaign and then it gets shifted a couple of years forward or back to fit with another campaign (happens often in mainline). I think this calendar should be discussed in the [Writers' Forum](https://r.wesnoth.org/f32) rather than a Github PR.

Reply:
I agree here. I don’t think a lunisolar calendar is goign to work, as it would require intercalation. This would leave the the option of a fully solar calendar (e.g. Gregorian Calendar, based on seasons) or a dully lunar calendar (e.g. Islamic Calendar, based on lunar cycles). A fully lunar calendar would drift from the season very quickly, which might not be ideal for the larger seasonal changes in Wesnoth and their effect on agriculture, so I’d personally think a solar calendar whose months alternate between 30 and 31 days (366 days, fully flips seasons every 487 years). I will be making a thread on this later.

Reply:
Irdya isn't Earth - as long as it's close enough that human characters behave as players would expect from their age, I don't see why an Irdyan year should be exactly an Earth year.

If there's an exact definition, it probably should be at least as long an Earth year - don't want to create a situation where an adult in Irdyan years is a child in Earth years, even though we're not intending to tell stories where that would be a concern.

Reply:
Maybe a calendar with 12 32-day months, which adds up to 372 years? I do not currently intend to change the calendar system, so I’m resolving this.